code/game/objects/items/melee/misc.dm

```diff
@@ -505,3 +505,82 @@
 	held_sausage.name = "[target.name]-roasted [held_sausage.name]"
 	held_sausage.desc = "[held_sausage.desc] It has been cooked to perfection on \a [target]."
 	update_icon()
+
+/obj/item/twohanded/required/electrostaff
+	icon = 'icons/obj/estaff.dmi'
+	icon_state = "electrostaff_0"
+	item_state = "electrostaff_0"
+	lefthand_file = 'icons/mob/inhands/weapons/estaff_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/estaff_righthand.dmi'
+	name = "riot suppression electrostaff"
+	desc = "A large quarterstaff, with massive silver electrodes mounted at the end."
+	force = 1
+	damtype = "fire"
```

Comment:
fire? Why not electric? 
iirc a fire suit may make this weapon useless, or maybe you intent for that?
Then again im not 100% if damtype set to fire looks at melee or fire armor...

Reply:
it does burn damage, that's what fire means. All melee damage goes through melee armor,